entity-service-api/src/main/proto/org/hypertrace/entity/data/service/v1/entity_data_service.proto

```diff
@@ -12,6 +12,7 @@ service EntityDataService {
   }
   rpc upsertEntities (Entities) returns (Empty) {
   }
+  rpc getAndUpsertEntities (Entities) returns (stream Entity) {}
```

Comment:
```suggestion
  rpc getAndUpsertEntities (Entities) returns (stream Entity) {
  }
```

Reply:
As rest of the methods are following that pattern.

Reply:
I think this is very minor. Will avoid one more commit and full build cycle for this :)